docs/source/contributor-guide/plugin_overview.md

```diff
@@ -64,25 +74,52 @@ of this could outweigh the benefits of running parts of the query stage natively
 
 ## Query Execution
 
-Once the plan has been transformed, any consecutive Comet operators are combined into a `CometNativeExec` which contains
-a serialized version of the plan (the serialization code can be found in `QueryPlanSerde`). When this operator is
-executed, the serialized plan is passed to the native code when calling `Native.createPlan`.
+Once the plan has been transformed, any consecutive native Comet operators are combined into a `CometNativeExec` which contains
+a protocol buffer serialized version of the plan (the serialization code can be found in `QueryPlanSerde`).
+
+Spark serializes the physical plan and sends it to the executors when executing tasks. The executors deserialize the
+plan and invoke it.
+
+When `CometNativeExec` is invoked, it will deserialize the protobuf plan and pass it into
```

Comment:
I think `CometNativeExec` doesn't deserialize the protobuf plan. It is deserialized in native.

Reply:
Yea.